stdlib/REPL/src/latex_symbols.jl

```diff
@@ -111,6 +111,7 @@ const latex_symbols = Dict(
     "\\impliedby" => "⟸",
     "\\to" => "→",
     "\\euler" => "ℯ",
+    "\\ohm" => "\u2126",
```

Comment:
Use `"Ω"` here to match the style of the other code.

Reply:
Resolved in 766bc0a.